packages/rs-drive-abci/src/execution/storage/store_reduced_platform_state/v0/mod.rs

```diff
@@ -0,0 +1,24 @@
+use crate::error::Error;
+use crate::platform_types::platform::Platform;
+use crate::platform_types::platform_state::PlatformState;
+use dpp::serialization::{PlatformSerializable, ReducedPlatformSerializable};
+use dpp::version::PlatformVersion;
+use drive::query::TransactionArg;
+
+impl<C> Platform<C> {
+    pub(super) fn store_reduced_platform_state_v0(
+        &self,
+        state: &PlatformState,
+        transaction: TransactionArg,
+        platform_version: &PlatformVersion,
+    ) -> Result<(), Error> {
+        self.drive
+            .store_reduced_platform_state_bytes(
```

Comment:
not sure it makes sense to create separate `store_reduced_platform_state_bytes`, I would just put that code here

Reply:
it follows similar logic like in `store_platform_state`